__tests__/app.test.js

```diff
@@ -0,0 +1,14 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/index.js");
+
+describe.only("GET /api/topics", () => {
+  test("calls this endopoint and sees if it is an array or not", () => {
```

Comment:
We need more tests than this. This is not checking the keys in the objects for example

Reply:
HI August
thank you for the comments. It looks like i hadnt pushed my changes before making a pull request. i have added them now.